src/backend/kubernetes_backend.py

```diff
@@ -860,3 +860,103 @@ def write_service_config(self, source_path: str, service_name: str, destination_
             _preload_content=False,
         )
         self.log.info(f"Finished unpacking config data for {service_name} to {destination_path}")
+
+    def copy_file_to_pod(self, pod_name, src_path, dest_path):
```

Comment:
I hate this function.

Also, I noticed testing this out that a tonne of things are way easier via `kubectl exec -it warnet-tank-<index>`. This is what I was missing since we dropped compose; hop into the container with a single command.

It made me think though, I think a tonne of our functionality could be re-written to use the k8s client object directly, and end up a lot neater...

Reply:
You could also look in to using the sidecar image I made for simln: https://hub.docker.com/r/pinheadmz/sidecar. One benefit of this is that it is paired with a shared volume so if the container restarts, the data is still there.

Reply:
Interesting... will investigate